yb-voyager/cmd/live_migration.go

```diff
@@ -57,8 +59,30 @@ func streamChanges() error {
 
 func handleEvent(event *tgtdb.Event) error {
 	log.Debugf("Handling event: %v", event)
-
-	// TODO: Convert values in the event to make it suitable for target DB.
+	tableName := event.TableName
+	if sourceDBType == "postgresql" && event.SchemaName != "public" {
+		tableName = event.SchemaName + "." + event.TableName
+	}
+	// preparing value converters for the streaming mode
+	tableSchema := dbzm.GetTableSchema(tableName, exportDir)
+	columnToSchema := make(map[string]dbzm.Schema)
+	for _, column := range tableSchema.Columns {
+		columnToSchema[column.ColName] = column
+	}
+	for column, value := range event.Fields {
```

Comment:
We might have to do this for event.Key as well, I guess? (what if the primary key is one of the types that needs transformation)

Reply:
yes, completely missed it. I remember it in the starting but while working missed that. Thanks for catching that. I will add it.